src/App.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import logo from './logo.svg';
-import { Counter } from './features/counter/Counter';
+
 import './App.css';
+
```

Comment:
this empty space is not necessary

Reply:
Will be removing empty spaces on next commit